wordpress-mobile/WordPress-iOS
# Add announcement for Stats Revamp

Addresses https://github.com/wordpress-mobile/WordPress-iOS/issues/18624

This PR adds the UI for the Stats Revamp's feature announcement. It doesn't implement the functionality for showing the announcement, that will be done in a later PR.

To test:

1. In Xcode, open the app delegate and temporarily add a call to `showStatsRevampV2FeatureIntroduction()` to the end of the `applicationDidBecomeActive` method, like so: `WPTabBarController.sharedInstance().showStatsRevampV2FeatureIntroduction()`.
2. Open the app, tap Profile pic → App Settings → Debug Settings → turn "New Cards for Stats Insights" ON
3. Background the app
4. Foreground the app 
5. Notice that the feature announcement is shown
6. Compare the screen to the design shown in Figma (the one shown in the linked issue may be outdated)
    - The image that's shown on the announcement screen appears to have a faint light blue background. When I exported the image from Figma as a PDF for inclusion in the asset catalogue, its background appears to be light gray. I've left it as-is as I'm not sure what's going on and the different might be only very slight or unnoticeable. Let me know if you have any pointers here, otherwise I'm happy to leave it as-is.

## Regression Notes
1. Potential unintended areas of impact

This PR doesn't implement functionally for showing the announcement, so it's unlikely this can negatively impact other parts of the app.

8. What I did to test those areas of impact (or what existing automated tests I relied on)

Not applicable.

9. What automated tests I added (or what prevented me from doing so)

This PR is just UI-related, without functionality, so I haven't looked at adding tests. In a follow-up PR when the screen's functionality is implemented, I can reconsider tests. Let me know if you think I should add tests here.

PR submission checklist:
